src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -285,6 +285,21 @@ public SyntaxToken Lex(LexerMode mode)
         private SyntaxListBuilder _leadingTriviaCache = new SyntaxListBuilder(10);
         private SyntaxListBuilder _trailingTriviaCache = new SyntaxListBuilder(10);
 
+        private SyntaxListBuilder? _directiveTriviaCache;
+        private SyntaxListBuilder GetDirectiveTriviaCache()
```

Comment:
>GetDirectiveTriviaCache [](http://example.com/codeflow?start=34&length=23)

Please document the purpose of this API and the proper way to use it.

Reply:
Removed in iteration 15